arviz/stats/stats.py

```diff
@@ -88,8 +88,9 @@ def compare(
 
     Returns
     -------
-    A DataFrame, ordered from lowest to highest IC. The index reflects the key with which the
+    A DataFrame, ordered from best to worst model. The index reflects the key with which the
```

Comment:
lgtm! "best" and "worst" are sometimes loaded words -- is it very accepted that they are appropriate here? Would a note like "ordered from best to worst model  (measured by information criteria)." be more clear?

Reply:
Yeah, I share the same concern.